contracts/libraries/PublishingLogic.sol

```diff
@@ -89,12 +89,14 @@ library PublishingLogic {
             _profile.followModule = followModule;
         }
 
-        bytes memory followModuleReturnData = _initFollowModule(
-            profileId,
-            followModule,
-            followModuleData,
-            _followModuleWhitelisted
-        );
+        bytes memory followModuleReturnData = new bytes(0);
```

Comment:
I don't know if the compiler is not smart enough yet, but I tried removing this (and the other similar one that is below)
```suggestion
        bytes memory followModuleReturnData;
```
and I saved some gas.

From `1343974` to `1338147` in the gas reports, which is `5827`.

Reply:
wtf the optimizer is supposed to handle this, I left them in to be explicit :sob:

Reply:
Resolved now!